client/src/registerServiceWorker.ts

```diff
@@ -21,7 +21,7 @@ const isLocalhost = Boolean(
 export default function register() {
```

Comment:
You know, I don't think we actually need this file at all...

Reply:
It is used in index.tsx. Do we not need to call this anymore?

Reply:
i'm just going to merge and we can remove this file and the call if we want to